x/nexus/types/params.go

```diff
@@ -105,3 +114,20 @@ func validateChainMaintainerCheckWindow(i interface{}) error {
 
 	return nil
 }
+
+func validateConnectionRouter(i interface{}) error {
+	val, ok := i.(sdk.AccAddress)
+	if !ok {
+		return fmt.Errorf("invalid parameter type for ConnectionRouter: %T", i)
+	}
+
+	if len(val) == 0 {
+		return nil
+	}
+
+	if err := sdk.VerifyAddressFormat(val); err != nil {
```

Comment:
can enforce 32 bytes

Reply:
that's not enforced in cosmos anymore though, So I don't think we should either